ui-lib.css

```diff
@@ -530,9 +538,12 @@ header .logo {
 .acc-pane.acc-active h3:before {
   background-position: 0 3px;
 }
-.acc-pane.acc-active h3:hover {
-  text-decoration: none;
+
+.acc-pane.acc-active h3:hover:before,
+.acc-pane.acc-active h3:focus:before {
+  background-position: 0 -41px;
```

Comment:
Changes the icon on hover and focus of an already expanded panes to indicate the click behavior. This mirrors the current hover behavior on minimized panes. Minimized panes show a left pointing arrow without hover, and a downward pointing arrow on hover (to indicate the pane will expand on click). This addition leaves the previous down arrow shown on an expanded panel under most conditions, but now shows a left pointing arrow on hover and focus (to indicate the pane will shrink on click).